src/index.html

```diff
@@ -5,8 +5,361 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="defalit"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Great+Vibes&family=Nothing+You+Colid+Do&family=Roboto:ital,wght@0,500;0,700;1,500;1,700&family=Satisfy&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="moyo_logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a class="nav__list-link nav__list-link--is-active is-active" href="#apple">Apple</a>
```

Comment:
It's quite hard to read.
```suggestion
            <a 
              class="nav__list-link nav__list-link--is-active is-active" 
              href="#apple"
            >
              Apple
            </a>
```